src/main/java/io/quarkus/gizmo2/creator/ops/ObjectOps.java

```diff
@@ -67,61 +67,61 @@ protected Expr receiver() {
     }
 
     /**
-     * Perform an instance invocation.
+     * Perform an instance invocation of a method with 3 parameters.
      *
      * @param returnType the return type (must not be {@code null})
      * @param name the method name (must not be {@code null})
-     * @param arg0Type the first argument type (must not be {@code null})
-     * @param arg1Type the second argument type (must not be {@code null})
-     * @param arg2Type the third argument type (must not be {@code null})
+     * @param param0Type the first parameter type (must not be {@code null})
+     * @param param1Type the second parameter type (must not be {@code null})
+     * @param param2Type the third parameter type (must not be {@code null})
```

Comment:
I guess you're correct here; these are indeed the parameter types and not argument types.